go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/shipwright-io/operator
 
-go 1.20
+go 1.22
```

Comment:
side note: pls ensure that the operator can still be build when indicating a go module dependency of v1.22. For example, if the environment where the binary of this is using go v1.21, it might be a problem.

Reply:
For this branch - bumping go to 1.22 will definitely break things

Reply:
Reverted to go 1.20